WORKSPACE

```diff
@@ -89,7 +89,7 @@ http_archive(
 git_repository(
     name = "mediapipe",
     remote = "https://github.com/openvinotoolkit/mediapipe",
-    commit = "ac9ff003cd735f96a2e5501e32b2c941f6d4543d", # Updated CAPI calls (#21)
+    commit = "18eddf82f473ebafa3c017805d3049d29b5404f6", # Updated CAPI calls (#21)
```

Comment:
Commit will be bumped again after related MP fork repo MR will be merged to main

Reply:
As soon as I have approvals.